src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/StraxCoinstakeRule.cs

```diff
@@ -80,6 +80,13 @@ public override Task RunAsync(RuleContext context)
                     // Only OP_RETURN scripts are allowed in coinbase.
                     if (!txOut.ScriptPubKey.IsUnspendable)
                     {
+                        // Make an exception for refund outputs.
```

Comment:
how do we know they are refund outputs? Here basically all PayToPubkeyHash outputs are allowed 

also it goes against comment on line 67 `1. No outputs should be spendable (we could mandate that the PoS reward must be wholly contained in the coinstake, but it is sufficient that the coinbase outputs are unspendable)`

that comment should be updated I think

Reply:
see above.